### DISCUSSION TITLE
Library not reloading

### DISCUSSION THREAD

**Discussion by raphial** [martpie/museeks#618] [General]
Love the program! Running Linux Mint 20.1 based on Ubuntu.

For some reason my library will not update when new folders/song files are added. If I force a Reload from the menus, it still does not update. Any suggestions?

**Comment by martpie** [comment]
Folders are not automatically re-scan, you'll have to import your music again (already imported tracks won't be re-imported)

**Reply by raphial** [reply]
For clarification, you have to drag/drop the folder again into the Library settings?

Are you not able to drag/drop the new music files into the Library screen?

**Reply by martpie** [reply]
It has to be from the settings view indeed, it's not yet supported to import music from other views :) I believe there is a task somewhere for it, and I think I'll add it at some point.

**Reply by raphial** [reply]
I gotchya, ya that worked. If we can add the drag/drop feature into other views, that would be extremely helpful. And if we can have a scan feature! :)

**Reply by martpie** [reply]
One thing I'm planning to re-implement are "library folders" you could add in the settings view (we used to have it and I remove it when I worked on an improved import feature a looooong time ago.

Once we have that, we could imagine re-scanning the folders on start-up :)

**Reply by raphial** [reply]
Is there a way to add songs from a folder directly into a playlist? I have thousands of songs in my Library, it makes it difficult going through them all to make a playlist.

**Reply by martpie** [reply]
No, right now m3u files are imported on library scans, but no more. If you have a big library, it could be a good idea to manage your playlists with m3u files so you can import/export them to other players.

You can import/export m3u from Museeks itself.

**Reply by martpie** [reply]
Update: implementing the "Add music from any view" feature, see screencast on #606
